modules/datagrid/72/launch/added/launch/infinispan-config.sh

```diff
@@ -65,7 +65,7 @@ function prepareEnv() {
   unset TRANSPORT_LOCK_TIMEOUT
   unset MEMCACHED_CACHE
   unset DEFAULT_CACHE
-  
+
```

Comment:
These formatting things always seem to get introduced on my commits as well. Can you remove them? If we merge them it will make diffs a nightmare over time.